auto_py_to_exe/dialogs.py

```diff
@@ -40,7 +40,7 @@ def ask_file(file_type):
         file_path = askopenfilename(parent=root, filetypes=file_types)
     root.update()
 
-    if file_path == "":
+    if not isinstance(file_path, str) or file_path == "":
```

Comment:
Was this causing some issues? I haven't run into it myself.
Would `ask_files`, `ask_folder` and `ask_file_save_location` potentially suffer the same issue?

Reply:
@brentvollebregt  The problem is `ask_files` dialog returns empty tuple `()`, if user doesn't select a file or folder. `Eel` cannot transfer with web sockets a variable with tuple type, that's why it raises a runtime error described above :point_up: . I decided to fix it by checking type of the variable. Methods `ask_files`, `ask_folder` and `ask_file_save_location` may indeed have the same issue. I will try to resolve it!

Reply:
@brentvollebregt I found out a way to check file dialog response without type checking using the `bool()` operator convert. I described in comments how it works, for more information here is the [reference](https://docs.python.org/3/library/stdtypes.html#truth). Checked manually all test cases, works fine :ok_hand: